docs/ref/csrf.txt

```diff
@@ -85,11 +85,11 @@ You can acquire the token like this:
 .. code-block:: javascript
 
     function getCookie(name) {
-        var cookieValue = null;
+        const cookieValue = null;
```

Comment:
Needs to be a `let` (if we keep it), because we are assigning it below.

Reply:
The assignment isn't used, this declaration should be removed